### DISCUSSION TITLE
Custom publisher configuration error "unsupported properties: outputfolder"

### DISCUSSION THREAD

**Discussion by birbygit** [Keyfactor/ejbca-ce#527] [Q&A]
Hi all, I'm trying to configure a Certificate Sampler Custom Publisher.
I need to export all certificates inside a local folder  and next I've to manually move the folder contents to a FreeRadius instance that runs inside non accessible network (this is my only possibility).

I followed the basic configuration tutorial: https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/publishers-overview/custom-publishers/certificate-sampler-custom-publisher

When I fill the Properties section with the outputfolder, EJBCA issue this error: "Unsupported properties: outputfolder"

Thanks in advance.

**Comment by primetomas** [comment]
Thanks for the report. Unfortunately this is a regression in EJBCA 8.2.0 related to how properties are validated in the UI.
It may be possible to add in other ways, like CLI or database...but for the moment adding that in the UI is broken.
It's tracked internally as: https://jira.primekey.se/browse/ECA-12264
